### DISCUSSION TITLE
What is External Account Binding?

### DISCUSSION THREAD

**Discussion by dazinator** [polymind-inc/acmebot#435] [Q&A] [answered]
Hello, thank you for your hard work, I am just about to try and set this up.

I noticed the section on configuring "External Account Binding" in the docs - but I couldn't see an explanation on what that is for exactly, would you be able to elaborate?

**Comment by shibayan** [comment, marked as answer]
This is a mechanism for using ACME providers that requires user registration and authentication. Please refer to the documentation for details.

https://datatracker.ietf.org/doc/html/rfc8555#section-7.3.4
https://zerossl.com/documentation/acme/
https://dev.digicert.com/services-api/api-keys/acme-external-account-binding/

**Reply by dazinator** [reply]
Many thanks
